### DISCUSSION TITLE
Compatibility to Buderus Logatherm WLW286 A / WPM100

### DISCUSSION THREAD

**Discussion by git-developer** [emsesp/EMS-ESP32#1710] [Q&A] [answered]
I'd like to know if this project is compatible with a Buderus heatpump _Logatherm WLW286 A_ driven by a _Logatherm WPM100_ control unit.

I wasn't able to find out if the device has an EMS interface yet. The best I could find in a technical brochure ([PDF](https://productsde.buderus.com/buderus/productsde.buderus.com/broschueren/buderus-broschuere-einbaucheck-logatherm-wlw286-a-ar-de-102022.pdf)) is this diagram:
![grafik](https://github.com/emsesp/EMS-ESP32/assets/7452817/fa7c81b9-4635-452c-9e6f-696e0663d50c)

Looks like there's a CAN bus between heat pump and controller, and an option to add a hardware adapter for KNX.

I'd appreciate if someone could give a hint if these devices are compatible with EMS-ESP32, or how to integrate it into a smart home system otherwise.

**Comment by proddy** [comment]
@bbqkees  ?

**Reply by git-developer** [reply]
I guess chances are low to gain insights when @bbqkees has no idea.

**Comment by git-developer** [comment, marked as answer]
I think I found out some details that I'd like to share.

- It looks like this Buderus heat pump is made by Dimplex, not Bosch.
- The touch display controller _Logamatic WPM100_ is a relabelled _Dimplex WPM Touch_ device
- The connector labelled _BMS card_ is a slot that may hold 
    - either a _LWPM 410_ (article no. 339410) for Modbus RTU
    - or a _NWPM Touch_ (article no. 378800) for LAN or Modbus TCP
    - or a _KNX WPM_ (article no. 376350) for KNX
- Technical details for these extensions are described in the [Dimplex wiki](https://dimplex.atlassian.net/wiki/spaces/DW/overview?homepageId=2862055561)

If this correct, the EMS-ESP32 ist not compatible with this heat pump.

**Reply by git-developer** [reply]
These details have turned out to be correct. Integration of this heat pump into Smart Home systems is documented in [mobugen](https://github.com/git-developer/mobugen).
